docs/integrations/enterprise-connectors/source-workday.md

```diff
@@ -18,6 +19,70 @@ Airbyte's incubating Workday enterprise source connector currently offers the fo
 | SSL connection                | Yes        |
 | Namespaces                    | No         | 
 
+## Prerequisites
+* Workday tenant - The Organization ID for your Workday environment. This can be found by logging into your Workday account and going to My Account > Organization ID
+* Workday hostname - The endpoint for connecting into your Workday environment. This can be found by logging into your Workday instance and searching “Public Web Service” in the search bar and selecting the appropriate report. Use the ellipse (...) button to select **Web Service > View WSDL**
+
+* For REST API streams:
+	* Access token - An OAuth 2.0 access token for API client integrations. More information and instructions can be found in the Workday community documentation for your environment about creating and registering a Workday API Client. When registering the API Client for Airbyte, you can use https://cloud.airbyte.com/auth_flow for the Redirection URI field if you are using Airbyte Teams. If you are using Self-Managed Enterprise, you can replace cloud.airbyte.com with the URL of your Airbyte deployment.
+
+* For Report-as-a-Service (RaaS) streams:
+	 * Workday username and password - A user account that has the necessary permissions to access the reports you want to sync.
+	* Report IDs - Each report in Workday has a unique Report ID 
```

Comment:
I tried to stay consistent with this, other than some typos. The streams don't have any because they're generally 1-2 words but anything with a sentence structure I tried to conform to this